backend/remote/remote.go

```diff
@@ -249,14 +256,16 @@ func (rf *Factory) Create(address string) (types.Backend, error) {
 		return nil, err
 	}
 
-	rpc := rpc.NewClient(conn, r.closeChan)
-	r.IOs = rpc
+	remote := rpc.NewClient(conn, r.closeChan)
+	r.IOs = remote
 
 	if err := r.open(); err != nil {
+		remote.SetRWExitTrue()
+		remote.Close()
```

Comment:
if open is not success, should we do close?

Reply:
got it.. both are different

Reply:
correct, we need to close rpc connection in that case.